docs/ops/pooling/MaxPool_1.md

```diff
@@ -67,7 +139,10 @@
 
 **Inputs**:
 
-*   **1**: 4D or 5D input tensor. Required.
+*   **1**: 3D, 4D or 5D input tensor. Required.
+
+**Outputs**:
+  * **1**: Input shape can be either [N,C,H], [N,C,H,W] or [N,C,H,W,D]. Then the corresponding output shape will be [N,C,H_out], [N,C,H_out,W_out] or [N,C,H_out,W_out,D_out]
 
 **Mathematical Formulation**
 
```

Comment:
Mathematical Formulation section is very strange and useless. Please move your formulas from above and list formulas for different modes and rounding type here.